docs/pages/web/buttontoggle.tsx

```diff
@@ -0,0 +1,321 @@
+                name="Make sure that the ButtonToggle(s) in your application are consistently styled and placed. This should also apply to their sizing, maintaining uniformity throughout the experience."
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
+            type="do"
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description="Place the ButtonToggle(s) in a location where users would naturally expect to find them, taking into consideration the context. For instance, position it next to a related feature."
+            sandpackExample={
+              <SandpackExample
+                code={locationDo}
+                hideEditor
+                name="Place the ButtonToggle(s) in a location where users would naturally expect to find them, taking into consideration the context. For instance, position it next to a related feature."
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
+            type="do"
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description="If the ButtonToggle(s) trigger a significant action or irreversible change, it is recommended to include a confirmation, such as a toast message."
+            sandpackExample={
+              <SandpackExample
+                code={confirmationDo}
+                hideEditor
+                name="If the ButtonToggle(s) trigger a significant action or irreversible change, it is recommended to include a confirmation, such as a toast message."
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
+            type="do"
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <AccessibilitySection name={generatedDocGen?.displayName}>
+        <MainSection.Subsection
+          description={`
+When ButtonToggle text does not provide sufficient context about the ButtonToggle’s behavior, supply a short, descriptive label for screen-readers using \`accessibilityLabel\`.
+Texts like “Best“, “Views“, or “Save“ can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text with deeper context to replace the ButtonToggle text, like “Follow Ryan” or “Shop Wedding Invitations”.
+
+If ButtonToggle is used as a control button to show/hide a Popover-based component, we recommend passing the following ARIA attributes to assist screen readers:
+- \`accessibilityLabel\`: if present, read by screen readers read instead of the \`text\` prop.
+- \`accessibilityControls\`: informs the screen reader that ButtonToggle controls the display of an anchored Popover-based component. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
```

Comment:
let's use this links

accessibilityLabel : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label
 accessibilityControls : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

tweak the definition of accessibilityControls to the description in the link i gave you.

Reply:
DONE